### PR TITLE
Added audit trail comment validation

### PR DESCRIPTION
**Pull request by Gary-Griffiths-Methods**

### CODE REVIEW

**Review by andycyberian — COMMENTED** [review]
Have raised a couple of things that should simplify the code.

File: DFC.ServiceTaxonomy.ContentApproval/Drivers/ContentApprovalPartDisplayDriver.cs [hunk 1]
```diff
@@ -189,7 +181,163 @@ private static void PopulateViewModel(ContentApprovalPart part, ContentApprovalP
...
+        }
+
+        private static bool IsCommentEmpty(ContentApprovalPart part)
+        {
+            const string CommentFieldName = "Comment";
+            bool commentIsEmpty = true;
+
+            if (part != null && part.ContentItem != null && part.ContentItem.Content != null)
+            {
+                var stringContent = Convert.ToString(part?.ContentItem?.Content);
```
[the remark this answers is not in the document]
Comment (Gary-Griffiths-Methods) [reply DFC.ServiceTaxonomy.ContentApproval/Drivers/ContentApprovalPartDisplayDriver.cs]: Thanks Andy, that worked like a charm!

File: DFC.ServiceTaxonomy.ContentApproval/Models/AuditTrailPart.cs [hunk 2]
```diff
@@ -0,0 +1,9 @@
+﻿namespace DFC.ServiceTaxonomy.ContentApproval.Models
```
[the remark this answers is not in the document]
Comment (Gary-Griffiths-Methods) [reply DFC.ServiceTaxonomy.ContentApproval/Models/AuditTrailPart.cs:1]: Done

**Review by andycyberian — APPROVED** [review]
All looks good

**Pull request merged**